app/components/bali/side_menu/item/component.html.erb

```diff
@@ -0,0 +1,11 @@
+<li>
+  <%= render Bali::SideMenu::Link::Component.new(name: @name, href: @href, icon: @icon, **@options) %>
```

Comment:
This one should be at the component as renders_one

Reply:
I don't think so, when we use a slot, the slot is optional and has to be placed in the render component, this way we don't have to specify it when we're rendering the component.